src/client/utils/create-func-wrapper.js

```diff
@@ -0,0 +1,5 @@
+export default (func, thisObj) => {
```

Comment:
This is not actually what I mean. But I was wrong. We don't need arrow here. And we should rename it to `fnBind` and we need a comment that clarifies why we need it.